apps/api/src/campaign/campaign.service.ts

```diff
@@ -331,6 +351,15 @@ export class CampaignService {
     return await this.prisma.campaign.delete({ where: { id: campaignId } })
   }
 
+  async checkCampaignOwner(keycloakId: string, campaignId: string) {
+    const person = (await this.personService.findOneByKeycloakId(keycloakId)) as Person
```

Comment:
@Bunny303 Why do we need to type it explicitly here with `as Person`?

Reply:
Person for logged user is supposed to exist in the db, because it should be created on registration. It could not exist if request for creating person fails. That will leads to scnario where user could login (registration in Keycloak is successful), but person record won't exist in our db and all of the request that are not public will fail for him.
Throwing of error when person is not found will only alert the problem. May be we could think for solution that create the person in not exists in the database. 
It's explicitly typed, because findOneByKeycloakId could return null, but in this case it's not expected to be null.

Reply:
Ok, agree.